app/uk/gov/hmrc/cataloguefrontend/connector/ServiceDependenciesConnector.scala

```diff
@@ -127,24 +127,24 @@ class ServiceDependenciesConnector @Inject() (
       .get(url"$servicesDependenciesBaseUrl/api/repository-name?group=$group&artefact=$artefact&version=${version.toString}")
       .execute[Option[String]]
 
-  def getRepositoryModules(repositoryName: String)(implicit hc: HeaderCarrier): Future[Option[RepositoryModules]] = {
+  def getRepositoryModulesLatestVersion(repositoryName: String)(implicit hc: HeaderCarrier): Future[Seq[RepositoryModules]] = {
```

Comment:
Do the response types of the functions `getRepositoryModulesLatestVersion` and `getRepositoryModules` need to change? We're still only expecting a single result, so I think the connector should absorb the change.

Reply:
Done